### ISSUE TITLE
Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds 

### ISSUE THREAD

**Issue by A11Ybot-OP** [knative/ux#63]
Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.md-ellipsis)
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

Target application: Home - Knative - https://knative.dev/docs/

Element path: .md-header__topic:nth-child(1) > .md-ellipsis

Snippet: <span class="md-ellipsis"> Knative </span>

How to fix: 
Fix any of the following:
Element has insufficient color contrast of 4.35 (foreground color: #ffffff, background color: #1a7dc9, font size: 13.5pt (18px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Chrome version 97.0.4692.99

**Comment by A11Ybot-OP** [comment]
p.s there are 20 other colour -contrast issues
